index.js

```diff
@@ -608,6 +609,13 @@ async function newRegister(appkit, args) {
       }]);
     }, []);
     pipelines.sort((a, b) => a.name.toLowerCase().localeCompare(b.name.toLowerCase()));
+
+    try {
+      await appkit.http.post('', `${DIAGNOSTICS_API_URL}/v1/releasedhook`, plainType);
+      releasedHookEndpointExists = true;
+    } catch (err) {
```

Comment:
You might want to check here to make sure the error was a 404

Reply:
I’m trying to think what the better default would be in that situation.
1. `releasedHookEndpointExists` is always `true` unless the `/v1/releasedhook` returns a 404, or
1. `releasedHookEndpointExists` is always `false` unless the `/v1/releasedhook` returns a 2xx

I went with the latter, but I’m not sure which would be best. I guess maybe the question comes down to, what should happen if the `/v1/releasedhook` returns anything outside of a 2xx or 404? Maybe if the server returned a 5xx then we should assume that the endpoint at least exists, which means the server is on the new version. In that case the former would be better. What do you think?

Reply:
you can leave as-is if you prefer situation 2 honestly, it's up to you and I don't have a preference, but I usually throw an error out if something happened like it couldn't reach the TaaS backend because that would indicate an error with their network connection

Reply:
something like this https://github.com/akkeris/taas-plugin/blob/master/index.js#L39-L41

Reply:
I went with the first option. Thanks for helping me work it out!